### DISCUSSION TITLE
Espanso in Microsoft Store

### DISCUSSION THREAD

**Discussion by m0Re-p** [espanso/espanso#1255] [Feature requests and Ideas]
Hello,

It will be nice if you can add Espanso in to Microsoft Store. For example in office laptop usually can't install additional software except by Microsoft Store :)

Best regards 
m0Re-p

**Comment by federico-terzi** [comment]
Hey @m0Re-p,

Thank you for the feedback! I can't promise anything, but I'll investigate :)

Cheers :)
